chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/SurveillanceController.java

```diff
@@ -668,4 +680,19 @@ private void checkUploadedSurveillanceOwnership(Surveillance pendingSurv) {
 			} 
 		}
 	}
+	
+	private void updateAuthority(Surveillance surv){
```

Comment:
No; the purpose of this method is to handle when the UI passes in a null Surveillance authority.